crawler/config/centre.py

```diff
@@ -82,8 +97,12 @@
         "prefix": "RAND",
         "lab_id_default": "Randox",
         "backups_folder": f"{CENTRE_DIR_BACKUPS}/RAND",
-        "sftp_file_regex": r"^lw-randox-biocentre-box-.*\.csv$",
+        # "sftp_file_regex": r"^lw-randox-biocentre-box-.*\.csv$",
+        "sftp_file_regex_unconsolidated_surveillance": f"^lw-randox-biocentre-box-.*\.csv$",
+        "sftp_file_regex_consolidated_surveillance": r"^RAND-\d+\.csv$",
+        "sftp_file_regex_consolidated_eagle": r"^RXE\d+\.csv$",
         "sftp_root_read": "project-heron_randox",
+        # "file_names_to_ignore": [r"^lw-randox-biocentre-box-((\d)|(1\d)|20)-.*$"],
```

Comment:
Why comment this out? We do not want to process these older files as they were in the incorrect format.

Reply:
So these files will never be processed? Should we remove them instead?

Reply:
We did consider it but thought it might be too much effort at this time with having to remove the import records everywhere etc.

Reply:
Added this back